sdk/dns/arm-dns/package.json

```diff
@@ -8,11 +8,11 @@
     "node": ">=18.0.0"
   },
   "dependencies": {
-    "@azure/core-lro": "^2.5.4",
     "@azure/abort-controller": "^2.1.2",
```

Comment:
We should also make sure we upgrade to @azure/abort-controller 2.1.2 if not done already.  Many ARM packages are still on 1.0.0

Reply:
@kazrael2119 could you also create an issue for that in the SDK repo? we probably need to investigate a little about the impact. Thank you

Reply:
create an issue to track https://github.com/Azure/azure-sdk-for-js/issues/32732